argus/values.yaml

```diff
@@ -8,4 +8,6 @@ collectorVersion: 24200
 disableAlerting: false
 deleteDevices: false
 debug: false
-enableRBAC: true
+deployment:
```

Comment:
The default generated by Helm 2.5.1 is 
```
    {{- if .Values.nodeSelector }}
      nodeSelector:
{{ toYaml .Values.nodeSelector | indent 8 }}
    {{- end }}
```
I would prefer to skip the `.deployment` and just use `.Values.nodeSelector`. While I'm not a huge fan of the official helm style, the docs do recommend a flat style: https://docs.helm.sh/chart_best_practices/#flat-or-nested-values

Reply:
yeah, intresting to me I would nest it depending on file, blah blah. Works for me, it is my first edit to a helm chart.

Reply:
I would prefer it too to be quite honest. :) but that is what the community decided on, so I figured we go along with it.